buildSrc/src/main/java/org/elasticsearch/gradle/info/GlobalBuildInfoPlugin.java

```diff
@@ -106,6 +109,9 @@ private static File findRuntimeJavaHome(final File compilerJavaHome) {
         String runtimeJavaProperty = System.getProperty("runtime.java");
 
         if (runtimeJavaProperty != null) {
+            if (runtimeJavaProperty.contains("/")) {
```

Comment:
Note that this is temporary for testing this pull request. It will be removed before merging.